features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.listing/listing.js

```diff
@@ -17,6 +17,8 @@
  */
 
 function onRequest(context){
+    var page_data = {};
```

Comment:
Acknowledged and fixed with commit https://github.com/wso2/carbon-device-mgt/commit/3dc23c5708fbbd4dab63878611b30b5e7bed44d0